Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py

```diff
@@ -1103,3 +1128,56 @@ def test_key_not_found():
     result = Elasticsearch_v2.get_value_by_dot_notation(dictionary, key)
 
     assert result is None
+
+
+@pytest.mark.parametrize('limit, all_results, expected_context',
+                         [
+                             (1, True, [{'Name': 'index_name1',
+                                         'Status': 'open',
+                                         'Health': 'yellow',
+                                         'UUID': '1111',
+                                         'Documents Count': 2,
+                                         'Documents Deleted': 0},
+                                        {'Name': 'index_name2',
+                                         'Status': 'closed',
+                                         'Health': 'green',
+                                         'UUID': '2222',
+                                         'Documents Count': 40,
+                                         'Documents Deleted': 2
+                                         }
+                                        ]
+                              ),
+                             (1, False, [{'Name': 'index_name1',
+                                         'Status': 'open',
+                                          'Health': 'yellow',
+                                          'UUID': '1111',
+                                          'Documents Count': 2,
+                                          'Documents Deleted': 0}
+                                         ]
+                              )],
+                         ids=[
+                             "Test get indices statistics with a limit and all_results=True",
+                             "Test get indices statistics with a limit and all_results=False"]
+                         )
+def test_get_indices_statistics_command(mocker, limit, all_results, expected_context):
+    """
+    Given
+      - elastic search client
```

Comment:
the "given" should, in most cases, give explanation about the pre setup. 
so it is not the client it self, but the variables that are configured in ti an teir meaning.
In general, when using parametrize, it is better to add the docstring per case.

Reply:
Will ad it, thanks!